lib/developer_portal/app/views/developer_portal/docs.html.liquid

```diff
@@ -2,8 +2,8 @@
 
 <p>Use our live documentation to learn about the Echo API</p>
 
-{% cdn_asset /swagger-ui/2.2.10/swagger-ui.js %}
-{% cdn_asset /swagger-ui/2.2.10/swagger-ui.css %}
+{% cdn_asset /swagger-ui/3.24.3/swagger-ui-bundle.js %}
+{% cdn_asset /swagger-ui/3.24.3/swagger-ui.css %}
```

Comment:
I think there will be incompatibilities with our custom JS code to render the user keys and such